roofit/roofitcore/inc/RooConstraintSum.h

```diff
@@ -45,7 +45,7 @@ class RooConstraintSum : public RooAbsReal {
         const char* globalObservablesTag,
         bool takeGlobalObservablesFromData,
         bool cloneConstraints,
-        RooWorkspace * workspace = nullptr);
+        RooWorkspace * workspace);
```

Comment:
Is this needed ? It is a change in the interface for some potential users

Reply:
I would have avoided the bug that is fixed in this PR. And fortunately, this change doesn't break any existing code, because `RooConstraintSum::createConstraintTerm` was only introduced in this release development cycle!